diff -Nru tmpwatch-2.1/Makefile tmpwatch-2.1.new/Makefile
--- tmpwatch-2.1/Makefile	Thu Feb 10 00:51:40 2000
+++ tmpwatch-2.1.new/Makefile	Thu Feb 10 00:52:01 2000
@@ -15,10 +15,10 @@
 all: tmpwatch
 
 install: all
-	[ -d $(PREFIX)$(BASEDIR)/sbin ] || mkdir -p $(PREFIX)$(BASEDIR)/sbin
-	[ -d $(PREFIX)$(BASEDIR)/man/man8 ] || mkdir -p $(PREFIX)$(BASEDIR)/man/man8
-	$(INSTALL) -s -m 755 tmpwatch $(PREFIX)$(BASEDIR)/sbin/tmpwatch
-	$(INSTALL) -m 644 tmpwatch.8 $(PREFIX)$(BASEDIR)/man/man8/tmpwatch.8
+	[ -d $(DESTDIR)$(BASEDIR)/sbin ] || mkdir -p $(DESTDIR)$(BASEDIR)/sbin
+	[ -d $(DESTDIR)$(BASEDIR)/share/man/man8 ] || mkdir -p $(DESTDIR)$(BASEDIR)/share/man/man8
+	$(INSTALL) -s -m 755 tmpwatch $(DESTDIR)$(BASEDIR)/sbin/tmpwatch
+	$(INSTALL) -m 644 tmpwatch.8 $(DESTDIR)$(BASEDIR)/share/man/man8/tmpwatch.8
 
 clean:
 	rm -f tmpwatch
