--- tmpwatch-1.7/Makefile.old	Wed Jun 10 21:31:22 1998
+++ tmpwatch-1.7/Makefile	Tue May  4 02:26:29 1999
@@ -9,7 +9,7 @@
 	[ -d $(PREFIX)/usr/sbin ] || mkdir -p $(PREFIX)/usr/sbin
 	[ -d $(PREFIX)/usr/man/man8 ] || mkdir -p $(PREFIX)/usr/man/man8
 	install -s -m 755 tmpwatch $(PREFIX)/usr/sbin/tmpwatch
-	install -s -m 644 tmpwatch.8 $(PREFIX)/usr/man/man8/tmpwatch.8
+	install -m 644 tmpwatch.8 $(PREFIX)/usr/man/man8/tmpwatch.8
 
 clean:
 	rm -f tmpwatch
